mappings/servers/hvbc/metadata.json

```diff
@@ -0,0 +1,30 @@
+{
+    "id": "hvbc",
+    "name": "HVBC",
+    "description": "Practice Server For Legit player",
+    "addresses": [
+        "hvbc.club",
+    ],
+    "primaryAddress": "hvbc.club",
```

Comment:
I've noticed that this address does not respond to MC pings but `na.hvbc.club` does

So I'm guessing the the `na.` address is your primary address, and it is the one you should use here

Leave the `addresses` section as is